src/models/chains.rs

```diff
@@ -28,3 +29,11 @@ pub struct Theme {
     pub text_color: String,
     pub background_color: String,
 }
+
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct GasPriceOracle {
+    pub url: String,
+    pub gas_parameter: Option<String>,
+    pub gwei_factor: String,
```

Comment:
The `gwei_factor` is always present or does it have any default value ?

Reply:
Always present. Default value from the config service is `"1.000000000"`